library/src/main/java/com/github/pwittchen/reactivebus/library/Event.java

```diff
@@ -73,27 +93,27 @@ public int hashCode() {
     return Objects.hash(id, name);
   }
 
-  public static class Builder {
+  public final static class Builder {
     private String id = UUID.randomUUID().toString();
     private String name = "";
     private Serializable data;
 
-    public Builder setId(String id) {
+    public Builder id(String id) {
       this.id = id;
       return this;
     }
 
-    public Builder setName(String name) {
+    public Builder name(String name) {
       this.name = name;
       return this;
     }
 
-    public Builder setData(Serializable data) {
+    public Builder data(Serializable data) {
       this.data = data;
       return this;
     }
 
-    public Event build() {
+    public Event create() {
```

Comment:
This should still be called `build()` (you *build* with a builder).

Reply:
Fixed.